icon/iiss/icstate/accountCache.go

```diff
@@ -70,15 +71,20 @@ func (c *AccountCache) Clear() {
 }
 
 func (c *AccountCache) Reset() {
-	for _, account := range c.accounts {
-		if !account.IsEmpty() {
-			value := c.dict.Get(account.address)
+	for key, _ := range c.accounts {
+		addr, err := common.NewAddress([]byte(key))
+		if err != nil {
+			panic(errors.Errorf("Address convert error"))
+		}
+		value := c.dict.Get(addr)
 
-			if value == nil {
-				account.Clear()
-			} else {
-				account.Set(ToAccount(value.Object(), account.address))
-			}
+		if value == nil {
+			delete(c.accounts, key)
+		} else {
+			delete(c.accounts, key)
+			newac := newAccount(addr)
+			newac.Set(ToAccount(value.Object(), addr))
+			c.accounts[key] = newac
```

Comment:
Why delete cache? Any reason? You can overwrite it.

Reply:
I tried to Set() account which is already in the cache.
But some values are distorted during copy value.
I deleted the item and create new Account and assigned, then it worked

Reply:
account.Set(ToAccount(value.Object(), addr)) <= this doesn't work

Reply:
I debugged account type and changed Set() part as you said. Thank you